src/displayapp/screens/WatchFacePrideFlag.cpp

```diff
@@ -0,0 +1,337 @@
+#include "displayapp/screens/WatchFacePrideFlag.h"
+#include <lvgl/lvgl.h>
+#include "components/battery/BatteryController.h"
+#include "components/ble/BleController.h"
+#include "displayapp/screens/Symbols.h"
+
+using namespace Pinetime::Applications::Screens;
+
+namespace {
+  void EventHandler(lv_obj_t* obj, lv_event_t event) {
+    auto* screen = static_cast<WatchFacePrideFlag*>(obj->user_data);
+    screen->UpdateSelected(obj, event);
+  }
+
+  Pinetime::Controllers::Settings::PrideFlag GetNext(Pinetime::Controllers::Settings::PrideFlag prideFlag) {
+    const auto prideFlagAsInt = static_cast<uint8_t>(prideFlag);
+    Pinetime::Controllers::Settings::PrideFlag nextFlag;
+    if (prideFlagAsInt < 3) {
+      nextFlag = static_cast<Pinetime::Controllers::Settings::PrideFlag>(prideFlagAsInt + 1);
+    } else {
+      nextFlag = static_cast<Pinetime::Controllers::Settings::PrideFlag>(0);
+    }
+    return nextFlag;
+  }
+
+  Pinetime::Controllers::Settings::PrideFlag GetPrevious(Pinetime::Controllers::Settings::PrideFlag prideFlag) {
+    const auto prideFlagAsInt = static_cast<uint8_t>(prideFlag);
+    Pinetime::Controllers::Settings::PrideFlag prevFlag;
+    if (prideFlagAsInt > 0) {
+      prevFlag = static_cast<Pinetime::Controllers::Settings::PrideFlag>(prideFlagAsInt - 1);
+    } else {
+      prevFlag = static_cast<Pinetime::Controllers::Settings::PrideFlag>(3);
+    }
+    return prevFlag;
+  }
+
+  template <std::size_t N>
+  class PrideFlagData {
+  public:
+    constexpr PrideFlagData(const std::array<lv_color_t, N>& sectionColours,
+                            lv_color_t defaultTopLabelColour,
+                            lv_color_t labelTimeColour,
+                            lv_color_t defaultBottomLabelColour)
```

Comment:
more const I've overlooked until now

```suggestion
                            const lv_color_t defaultTopLabelColour,
                            const lv_color_t labelTimeColour,
                            const lv_color_t defaultBottomLabelColour)
```

Reply:
he brings up very good reasoning. I just followed along with the CLion suggestions as I thought those are best practice. Marking as resolved